### DISCUSSION TITLE
O3DE and New Comers (Beginners)

### DISCUSSION THREAD

**Discussion by IndieDev99** [o3de/o3de#17187] [General]
I found O3DE and I like only "Free and open Source" part of it. 
Creating a Project is even harder than any engine I have ever use.
Terrain tool is very different and hard to understand. 
Many bugs 
Slow loading
Z Up Coordinate system (confusing for Godot, Unity etc Users)

Still only reason to use this engine is Free and Open Source !!
 May be that is the reason very very few people are using it and no steam game I have seen made with this engine. 
Regards, 
 Sad and frustrated O3DE user.

**Comment by JailbreakPapa** [comment]
Yep, O3DE really needs stabilization, fast.

**Comment by FinnT730** [comment]
I feel like people forget that this is still having parts of CryEngine... and Lumberyard
So it will take a while, I think

**Reply by IndieDev99** [reply]
but you have to provide some tools so, users can make simple 3d games. we will wait for AAA quality tools for future

**Reply by FinnT730** [reply]
I mean... sure.. but there is a ton of legacy code likely still around, from the CryEngine days.
It is not something they can just remove and expect the core of the engine to remain working
